pages/common/ccache.md

```diff
@@ -0,0 +1,24 @@
+# ccache
+
+> C/C++ compiler cache.
+> More information: <https://ccache.dev/manual/4.9.html#_command_line_options>.
+
+- Show current cache [s]tatistics:
+
+`ccache -s`
+
+- Clear all [C]ache:
+
+`ccache -C`
+
+- Reset ([z]ero) statistics (but not cache itself):
+
+`ccache -z`
```

Comment:
We prefer using long options: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax

```suggestion
- Show current cache [s]tatistics:

`ccache --show-stats`

- [C]lear all cache:

`ccache --clear`

- Reset ([z]ero) statistics (but not cache itself):

`ccache --zero-stats`
```